src/vendors/__tests__/monitor.test.ts

```diff
@@ -14,6 +14,7 @@ describe('monitor', () => {
       parseAccount({
         address: 'oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe',
         liquid_balance: 756455428396,
+        /* eslint-disable @typescript-eslint/no-loss-of-precision */
```

Comment:
what's this situation here

Reply:
Oasismonitor is returning these numbers in JSON, but they lose precision when they are parsed as a number. 336982978187110627 rounds to 336982978187110660.

I had already warned @ennsharma and @aefhm for oasis-indexer: since oasisscan was using ROSE units formatted into string, there wasn't the same worry.

Reply:
:cry: